CMakeLists.txt

```diff
@@ -117,17 +118,24 @@ option(USE_GNUSTL "Get c++ library/headers from the gnustl.pc" OFF)
 
 set(TARGET_srt "srt" CACHE STRING "The name for the SRT library")
 
+set (DISPLAY_LOGGING DISABLED)
+
 if (NOT ENABLE_LOGGING)
 	set (ENABLE_HEAVY_LOGGING OFF)
-	message(STATUS "LOGGING: DISABLED")
 else()
 	if (ENABLE_HEAVY_LOGGING)
-		message(STATUS "LOGGING: HEAVY")
+		set (DISPLAY_LOGGING HEAVY)
 	else()
-		message(STATUS "LOGGING: ENABLED")
+		set (DISPLAY_LOGGING ENABLED)
+	endif()
+
+	if (ENABLE_HAICRYPT_LOGGING)
+		list (APPEND DISPLAY_LOGGING HAICRYPT)
 	endif()
 endif()
 
+message(STATUS "LOGGING: ${DISPLAY_LOGGING}")
```

Comment:
The result would be a bit confusing:

- DISABLED;HAICRYPT
- ENABLED;HAICRYPT
- HEAVY;HAICRYPT

Reply:
That actually is the intent - so that it's visible that HAICRYPT logging is on. This is considered kinda possibly dangerous for the security.

Reply:
My thought is that `DISABLED;HAICRYPT` is confusing for a regular contributor.
This means that generally, the logging is disabled except for the HAICRYPT module.
CORE LOGGING: ENABLED
HAICRYPT LOGGING: ENABLED

Or print a hint&warning if HAICRYPT logging is enabled:
"HAICRYPT LOGGING: ENABLED - consider turning off for security reasons"

Reply:
ok, something like that

Reply:
For haicrypt logging there already is a loud warning, so I'll simply remove it. This `DISPLAY_LOGGING` is to be used later with thread logging, so it would then display `ENABLED; THREAD`.

Note that with disabled logging no other logging type can be enabled - at least this is the intent.

Reply:
> This `DISPLAY_LOGGING` is to be used later with thread logging, so it would then display `ENABLED; THREAD`. 

Can we postpone this change for that PR then? Because at the moment it only adds one extra line and additional variable without a good reason.

Reply:
Maybe you are right. Let's leave it for now. Thread logging is in another branch, we can refresh it later.